components/playback/controls/YAxisControls.vue

```diff
@@ -177,10 +228,94 @@ export default {
         this.y_axis_zoom_in = "Zoom-In";
       }
     },
+    zoom_controls: function () {
+      this.$bvModal.show("y-axis-controls-settings");
+      this.controls = !this.controls;
+    },
+    y_axis_controls_commit: function (new_range) {
+      this.$bvModal.hide("y-axis-controls-settings");
+      const default_y_axis_zoom_idx = 0;
+      const new_y_zoom_levels = [
+        { y_min: new_range.y_min, y_max: new_range.y_max },
+        { y_min: 0, y_max: 200 },
+        { y_min: 25, y_max: 150 },
+        { y_min: 50, y_max: 150 },
+        { y_min: 50, y_max: 140 },
+        { y_min: 50, y_max: 130 },
+        { y_min: 55, y_max: 120 },
+        { y_min: 60, y_max: 120 },
+        { y_min: 60, y_max: 110 },
+        { y_min: 65, y_max: 110 },
+        { y_min: 70, y_max: 110 },
+        { y_min: 70, y_max: 105 },
+        { y_min: 75, y_max: 105 },
+        { y_min: 80, y_max: 105 },
+        { y_min: 85, y_max: 105 },
+        { y_min: 90, y_max: 105 },
+        { y_min: 94, y_max: 102 },
+        { y_min: 96, y_max: 100 },
+      ];
```

Comment:
This code should get the y_zoom_levels from the store instead of hard coding them here, it should then insert the new user zoom level into the list at the appropriate location instead of putting it always at the beginning.

The way it is now would not give the desired behavior if, for example, the user entered a zoom level of (96,102) the custom value should be placed between (94, 102) and (96, 100).

Reply:
Please share the details as what has to be done if the user entered range is outside the scope of the y_zoom_levels 
as well ? Kindly let me what has to be done, if y_zoom_levels are not ordered in ascending order.

Reply:
There is no scope of values for the default y_zoom_levels, as long as the business logic for the yaxis zoom controller enforces its rules any value the user enters is a valid zoom level.  All this code needs to do is place the values the user entered into an appropriate location in the list and then take that value out of the list once the user presses the +/- button. The list should be created in order and that order should not be changed by this code other than to insert/remove a single user entered value. There should never be more than 1 user defined value in this list at a time.

If the user entered the min/max allowed, (0,10000) then it would be placed at the top of the list since it is more zoomed out than the default minimum zoom and they can't zoom out any more than that.

Reply:
@Jason Person @Eli Fine  from my observation the y_axis_zoom_integration is very confusing and ambiguous, so my suggestion is I have done the best feasible solution of updating the y_zoom_levels of inserting as discussed in the Asana card and its complete as per my observation. https://app.asana.com/0/1152302614804188/1200062366486904/f  . If your mind keeps changing and the position of insertion and the requirement keeps modifying after the PR, I think its better you update the function 'y_axis_controls_commit' as per the new requirement and approve the same. I am fine with the same. From my side your comments https://github.com/CuriBio/mantarray-frontend-components/pull/200#discussion_r608825996, I don't understand how could it possible be implemented. So I feel you are the best person as you seem to have very good idea of how things work. So I request you to change as per newly modified  requirement.

Reply:
Kindly also take into to consideration if user entered value is (94, 102) then should he have to have duplicates or no insertions etc, and all forms of situation please do consider and implement the same.